src/components/AccountSecurity/MyOrganization.tsx

```diff
@@ -0,0 +1,348 @@
+import React, { Component } from 'react';
+import Alert from 'react-bootstrap/Alert';
+import { withAlert } from 'react-alert';
+import CheckSVG from '../../static/images/check.svg';
+import getServerURL from '../../serverOverride';
+
+interface Props {
+  name: string,
+  organization: string,
+  alert: any
+}
+
+interface State {
+ personName: string,
+ personEmail: any,
+ personRole: any,
+ isInEditMode: boolean,
+ editedPersonEmail: string,
+ editedPersonName: string,
+ editedPersonRole: any,
+ memberArr: any,
+ showPopUp: boolean,
+ numInvitesSent: any,
+ numInEditMode: any,
+ buttonLoadingState: boolean,
+}
+
+class MyOrganization extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      personName: '',
+      personEmail: '',
+      personRole: '',
+      isInEditMode: false,
+      editedPersonEmail: '',
+      editedPersonName: '',
+      editedPersonRole: '',
+      memberArr: [],
+      showPopUp: false,
+      numInvitesSent: 0,
+      numInEditMode: 0,
+      buttonLoadingState: false,
+    };
+    this.editButtonToggle = this.editButtonToggle.bind(this);
+    this.getEmailCell = this.getEmailCell.bind(this);
+    this.getNameCell = this.getNameCell.bind(this);
+    this.getRoleDropDown = this.getRoleDropDown.bind(this);
+    this.saveEdits = this.saveEdits.bind(this);
+    this.addMember = this.addMember.bind(this);
+    this.deleteMember = this.deleteMember.bind(this);
+    this.renderSuccessPopUp = this.renderSuccessPopUp.bind(this);
+    this.saveMembersBackend = this.saveMembersBackend.bind(this);
+  }
+
+  addMember(e) {
```

Comment:
sorry another comment about arrow functions, but also add type definitions. e is type event and the return is void i think
addmember = (e) => {